### ISSUE TITLE
Disabled listbox item accepts selection

### ISSUE THREAD

**Issue by danaki** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#1262]
Thank you for such a great project!

Im trying to make toggled buttons like so:
```
<ListBox Style="{StaticResource MaterialDesignToolToggleListBox}" SelectedIndex="0" Margin="20,0,0,0">
    <ListBoxItem>Pause</ListBoxItem>
    <ListBoxItem IsEnabled="False">Unpause</ListBoxItem>
    <ListBoxItem>Finish</ListBoxItem>
</ListBox>
```

Notice that 2nd list item is actually disabled. It's grayed out but still accepts clicks and can be selected. I believe it's a bug.

**Comment by Tide** [comment]
I tested your code and it looks like that is the expected behavior.  The selected item is disabled so in your code you can say if the item is disabled then do nothing.  I do not think you can stop the listbox from having a selected item without writing a custom ListBox or at least using a custom style to control the look in those situations.

**Comment by Keboo** [comment]
Fixed by #1312
